Makefile

```diff
@@ -155,6 +155,12 @@ dist_release:
 # additional sanity checks and file movement.
 build_dist_release:
 	make x-build-dist
+	# Reduce binary size by compressing binaries.
+	# Currently errors with `Couldn't find DIE referenced by DW_AT_abstract_origin`
+	# dwz ${CARGO_TARGET_DIR}/release/tikv-server
+	dwz ${CARGO_TARGET_DIR}/release/tikv-ctl
```

Comment:
em, why tikv-ctl can use dwz but tikv-server can't?

Reply:
I am not sure!

Reply:
seems tikv-ctl is already small

Reply:
Agreed :-P

Reply:
```
make[2]: Leaving directory '/home/hoverbear/git/tikv'
# Reduce binary size by compressing binaries.
# Currently errors with `Couldn't find DIE referenced by DW_AT_abstract_origin`
dwz /home/hoverbear/git/tikv/target/release/tikv-server
dwz: /home/hoverbear/git/tikv/target/release/tikv-server: Couldn't find DIE referenced by DW_AT_abstract_origin
make[1]: *** [Makefile:160: build_dist_release] Error 1
make[1]: Leaving directory '/home/hoverbear/git/tikv'
make: *** [Makefile:149: dist_release] Error 2
```